src/components/header/header.jsx

```diff
@@ -21,18 +26,46 @@ const Header = () => {
 			</div>
 			<input type='search' className='text-field' placeholder='Search For Games' />
 			<div className='navbar-cta'>
-				<NavLink to='/wishlist' className='badge-holder flex-column'>
-					<HeartIcon className='header-icon' />
+				{user && (
+					<>
+						<NavLink to='/wishlist' className='badge-holder flex-column'>
+							<HeartIcon className='header-icon' />
 
-					{wishlist && wishlist.length > 0 ? <div className='badge-icon'>{wishlist.length > 9 ? '9+' : wishlist.length}</div> : <></>}
-				</NavLink>
+							{wishlist && wishlist.length > 0 ? (
+								<div className='badge-icon'>{wishlist.length > 9 ? '9+' : wishlist.length}</div>
+							) : (
+								<></>
+							)}
+						</NavLink>
 
-				<NavLink to='/myCart' className='badge-holder flex-column'>
-					<ShoppingCartIcon className='header-icon' />
-					{totalCartQuantity > 0 ? <div className='badge-icon'>{totalCartQuantity > 9 ? '9+' : totalCartQuantity}</div> : <></>}
-				</NavLink>
+						<NavLink to='/myCart' className='badge-holder flex-column'>
+							<ShoppingCartIcon className='header-icon' />
+							{totalCartQuantity > 0 ? <div className='badge-icon'>{totalCartQuantity > 9 ? '9+' : totalCartQuantity}</div> : <></>}
+						</NavLink>
+					</>
+				)}
 				<div onClick={toggleTheme}>{theme === 'dark' ? <SunIcon className='header-icon' /> : <HalfMoonIcon className='header-icon' />}</div>
-				<button className='btn btn-link'>Hello, Login</button>
+				{user != null ? (
+					<div className='badge-holder'>
+						<img
+							className='avatar avatar-sm'
```

Comment:
Suggestion if possible to add alt tag in the image field.

Reply:
Yep will add thanks